docs/fundamentals/code-analysis/quality-rules/ca1869.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "CA1869: Cache and reuse 'JsonSerializerOptions' instances"
+description: "Learn about code analyzer rule CA1869 - Cache and reuse 'JsonSerializerOptions' instances"
+ms.date: 08/21/2023
+ms.topic: reference
+f1_keywords:
+ - CA1869
+ - AvoidSingleUseOfLocalJsonSerializerOptions
+helpviewer_keywords:
+ - CA1869
+author: Jozkee
+ms.author: dacantu
+---
+
+# CA1869: Cache and reuse 'JsonSerializerOptions' instances
+
+|                                     | Value                                  |
+| ----------------------------------- |----------------------------------------|
+| **Rule ID**                         | CA1869                                 |
+| **Category**                        | [Performance](performance-warnings.md) |
+| **Fix is breaking or non-breaking** | Non-breaking                           |
+| **Enabled by default in .NET 7**    | No                                     |
+
+## Cause
+
+A local instance of <xref:System.Text.Json.JsonSerializerOptions> is used once as the `options` argument of a <xref:System.Text.Json.JsonSerializer.Serialize> or <xref:System.Text.Json.JsonSerializer.Deserialize> call.
+
+## Rule description
+
+Using a local instance of <xref:System.Text.Json.JsonSerializerOptions> for serialization or deserialization can substantially degrade the performance of your application if your code executes multiple times since System.Text.Json internally caches serialization releated metadata into the provided instance.
```

Comment:
```suggestion
Using a local instance of <xref:System.Text.Json.JsonSerializerOptions> for serialization or deserialization can substantially degrade the performance of your application if your code executes multiple times since System.Text.Json internally caches serialization-related metadata into the provided instance.
```